File: python/fixStringCase.dart
String solve(String s) {
  int smalls = 0;
  int caps = 0;

  for (int i = 0; i < s.length; i++) {
    if (s.codeUnitAt(i) <= 90)
      caps++;
    else
      smalls++;
  }

  if (caps > smalls)
    return s.toUpperCase();
  else
    return s.toLowerCase();
}

File: python/fixStringCase.py
def solve(s):
    caps, smalls = 0, 0
    
    for i in s:
        if ord(i) <= 90:
            caps += 1
        else:
            smalls += 1
            
    if caps > smalls:
        return s.upper()
    else:
        return s.lower()
